apps/ui/src/components/Footer.tsx

```diff
@@ -33,43 +34,44 @@ interface LinkType {
 	href?: string;
 }
 
-const pcglLinks: LinkType[] = [
-	{
-		name: 'Contact Us',
-	},
-	{
-		name: 'Policies & Guidelines',
-	},
-	{
-		name: 'Help Guides',
-	},
-	{
-		name: 'Controlled Data Users',
-	},
-	{
-		name: 'PCGL Website',
-	},
-	{
-		name: 'Data Platform',
-	},
-];
-
-const policiesConditionsLinks: LinkType[] = [
-	{
-		name: 'Privacy Policy',
-	},
-	{
-		name: 'Terms & Conditions',
-	},
-	{
-		name: 'Publication Policy',
-	},
-];
-
 const FooterComponent = () => {
+	const { t } = useTranslation();
```

Comment:
Kind of a headache to refactor (lots of instances) but I'd recommend changing this to `const { t: translate } = useTranslation();` and updating usage to `name: translate('links.contact')`

Reply:
EVEN BETTER  `const { t: metamorphosiseTranslatableText } ...` 😂 

I agree, `translate` is much better! Refactoring. 😎